spec/models/keyword_spec.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Keyword, type: :model do
+  describe 'Init' do
+    let(:keyword) { Fabricate(:keyword) }
+
+    it 'has name' do
+      expect(keyword.name).not_to be_nil
+    end
```

Comment:
I would not recommend testing the attributes of an object.
The main reason is that attributes are a mean to achieve something, but they are not an end. 
It is a bit similar for the "model associations", we could write beautiful tests to assert the one-to-many or many-to-many associations between models, but the tests would be similar to the code in the model (see [Tautological Tests](https://randycoulman.com/blog/2016/12/20/tautological-tests/)), so it is better to have tests that focus on the logic and that relies on these associations to test some business logic.


In case you really want to test it, I would recommend going 1 step further to be closer to testing business logic:

```suggestion
    it 'exposes its name' do
      keyword = Fabricate(:keyword, name: 'A keyword name')
      
      expect(keyword.name).to eq('A keyword name')
    end
```

but again, my opinion is that it is not needed 😇

Reply:
I agree with you, this test is redundant of the init functionality of the object but I would like to keep it here so that the `keyword` model has some test that I can build on in the future features. I think of it as a starting point for other tests for this model.

Taking you suggestion, I will update the test to not use Fabricator for the `expose` test, as we are testing the `init` function and also the `read` permission.

Reply:
Ok. 

Small note: `read permission` => It's usually called `visibility` (attributes visibility, method visibility, ... Either public, protected or private).

Let's keep `permission` for requests params and authorization ;-)